next.config.js

```diff
@@ -42,23 +42,25 @@ module.exports = withTM({
       //   use: ['babel-loader'],
       },);
   
-      config.plugins.push(
-        new MonacoWebpackPlugin({
-          languages: [
-            "json",
-            "markdown",
-            "css",
-            "typescript",
-            "javascript",
-            "html",
-            "graphql",
-            "python",
-            "scss",
-            "yaml"
-          ],
-          filename: "static/[name].worker.js"
-        })
-      );
+      if (!isServer) {
```

Comment:
I had to add this because `npm run build` wasn't working. Found it here: https://github.com/vercel/next.js/issues/31692